config/7.7.0/pim/pim.yaml

```diff
@@ -0,0 +1,160 @@
+## KIE PIM BEGIN
+pim:
+  deploymentConfigs:
+    - metadata:
+        name: "[[.ApplicationName]]-process-migration"
+        labels:
+          app: "[[$.ApplicationName]]"
+          application: "[[$.ApplicationName]]"
+          service: "[[.ApplicationName]]-process-migration"
+      spec:
+        replicas: 1
+        selector:
+          deploymentConfig: "[[.ApplicationName]]-process-migration"
+        strategy:
+          type: Recreate
+        triggers:
+          - type: ImageChange
+            imageChangeParams:
+              automatic: true
+              containerNames:
+                - "[[.ApplicationName]]-process-migration"
+              from:
+                kind: ImageStreamTag
+                namespace: openshift
+                name: "[[.PIM.Image]]:[[.PIM.ImageTag]]"
+          - type: ConfigChange
+        template:
+          metadata:
+            name: "[[.ApplicationName]]-process-migration"
+            labels:
+              app: "[[.ApplicationName]]"
+              application: "[[.ApplicationName]]"
+              deploymentConfig: "[[.ApplicationName]]-process-migration"
+              service: "[[.ApplicationName]]-process-migration"
+          spec:
+            serviceAccountName: "[[$.ApplicationName]]-[[$.Constants.Product]]svc"
```

Comment:
It doesn't need a serviceAccount.

Reply:
The `serviceAccount` is required by the `ose-cli` init-container to execute `oc get dc`.